swagger.json

```diff
@@ -1821,6 +1821,69 @@
             }
           }
         },
+        "/fleet/vehicles/stats": {
+            "parameters": [
+                {
+                  "$ref": "#/parameters/accessTokenParam"
+                },
+                {
+                  "name": "startMs",
+                  "description": "Time in Unix epoch milliseconds for the start of the query (cannot exceed 1 hour)",
+                  "required": true,
+                  "in": "query",
+                  "type": "integer",
+                  "format": "int64"
+                },
+                {
+                  "name": "endMs",
+                  "description": "Time in Unix epoch milliseconds for the end of the query (cannot exceed 1 hour)",
+                  "required": true,
+                  "in": "query",
+                  "type": "integer",
+                  "format": "int64"
+                },
+                { 
+                  "name": "filter",
+                  "description": "Comma-separated list of stat types. Options are engineState, auxInput1, and auxInput2.",
```

Comment:
I think this should be a required query parameter, since we want to avoid returning all timeseries data if this API gets larger.